tests/jerry/es.next/module-circular-02.mjs

```diff
@@ -0,0 +1,78 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as o from "module-circular-01.mjs"
+
+if (o.early != "Loaded")
```

Comment:
Is it possible to not run a test in this directory? Not really a problem if not, just asking. Perhaps adding a bad extension, e.g.  `._mjs`